spec/requests/keywords_controller_spec.rb

```diff
@@ -31,4 +31,16 @@
       end
     end
   end
+
+  describe 'GET #show' do
+    context 'given a valid id' do
+      it 'returns http success' do
```

Comment:
Let's add assertion for page content

Reply:
Good suggestion! Added one more assertion for page content
Fixed in [`5f004ca` (#47)](https://github.com/suho/google-search-ruby/pull/47/commits/5f004cae025caf7b86ce910239bd5478850fb86a)